Sources/Corvus/Endpoints/CRUD.swift

```diff
@@ -2,23 +2,33 @@ import Vapor
 
 /// A class that contains Create, Read, Update and Delete functionality for a
 /// generic type `T` conforming to `CorvusModel` grouped under a given path.
-public final class CRUD<T: CorvusModel>: Endpoint {
+public class CRUD<T: CorvusModel>: Endpoint {
 
     /// The route path to the parameters.
     let pathComponents: [PathComponent]
 
     /// A property to generate route parameter placeholders.
     let parameter = Parameter<T>()
 
-    /// Indicates wether soft delete should be included or not.
+    /// Indicates whether soft delete should be included or not.
     let useSoftDelete: Bool
     
     /// Initializes the component with one or more route path components.
     ///
     /// - Parameter pathComponents: One or more `PathComponents` identifying the
     /// path to the operations defined by the `CRUD` component.
     /// - Parameter softDelete: Enable/Disable soft deletion of Models.
-    public init(_ pathComponents: PathComponent..., softDelete: Bool = true) {
+    public init(_ pathComponents: PathComponent..., softDelete: Bool = false) {
+        self.pathComponents = pathComponents
+        self.useSoftDelete = softDelete
+    }
+    
+    /// Initializes the component with multiple route path components.
+    ///
+    /// - Parameter pathComponents: Multiple `PathComponents` identifying the
+    /// path to the operations defined by the `CRUD` component.
+    /// - Parameter softDelete: Enable/Disable soft deletion of Models.
+    public init(_ pathComponents: [PathComponent], softDelete: Bool = false) {
```

Comment:
I think we can make this initializer internal as it is only used within the auth modifier and this could reduce the API surface.